pyproject.toml

```diff
@@ -164,6 +164,7 @@ skip_covered = true
 exclude_lines = [
     "pragma: no cover",
     "if TYPE_CHECKING:",
+    ": \\.\\.\\.$",
```

Comment:
just asking. why are we ignoring  these lines?
maybe add a coment to explain why.

Reply:
I added a comment